docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -63,53 +62,101 @@ spec:
+
 Here are specific examples for how to clean up and use the YAML for certain resource kinds:
 
-* For any kind of resource, the `namespace`, `creationTimestamp`, `generation`, `resourceVersion`, `uid` and `ownerReferences`, metadata fields should be removed as well as the `status` section.
+* For any kind of resource, the `namespace`, `creationTimestamp`, `generation`, `resourceVersion`, `uid`, `ownerReferences` and `finalizers` metadata fields should be removed as well as the `status` section.
 * For *Application* resources the `metadata.name` and `spec.displayName` fields should contain variable references.
 * For *Component* resources:
-** The following annotations should be removed:
-*** `build.appstudio.openshift.io/status`
+** The following deprecated annotations should be removed:
 *** `image.redhat.com/image`
 ** The `spec.containerImage` field should be removed.
 ** The following fields should probably contain variable references:
 *** `spec.application`
 *** `spec.componentName`
 *** `source.git.revision`
+** The `build.appstudio.openshift.io/status` annotation is in place to make the UI present the component's pipeline as customized. Its not mandatory if you can ignore the pipeline status being missrepresented in the UI. Note that this annotation's value needs to contain a reference to a merged PR that added the PaC pipeline files. There is no harm in having multiple components reference the same PR, as long as the `pac.component.appstudio.openshift.io/finalizer` finalizer is not added to the component's finalizers list.
+* For *ImageRepository* resources:
+** The labels referring to the owning component and application should probably contain variable references.
+** To allow for correct ownership configuration between Component and ImageRepositorey resources, every componet that has its built image pushed to the default registry organisation that is managed automatically by {ProductName}, needs to have a Dedicated ImageRepositorey resource. This implies each component has its own container image repository.
+** To meet the above requirement the `spec.image.name` field should contain a variable reference.
+* For *IntegrationTestScenario* resources:
+** Make sure you query for the `v1beta2` version of those resources, and that you specify that version in your template.
+** The `spec.application` field should probably contain variable references.
+* For *ReleasePlan* resources:
+** The `spec.application` field should probably contain variable references.
```

Comment:
Can we just suggest what users should do here instead of stating that the fields should probably contain variable references?

Reply:
No, becasue we cannot assume that in all cases the users would be creating a ReleasePlan that is referring to an application that is being created by the same template.